### PR TITLE
Use DecapCMS for most of the website content

### PR DESCRIPTION
**Pull request by manuhabitela**
Following up on https://github.com/numerique-gouv/lasuite-landingpage/pull/39.

The idea here is to use DecapCMS for service landing pages, for the homepage, for the "communs" page, and for basic content pages like legal or privacy pages.

The PR is a bit much I'm so sorry. DecapCMS required a bit more work than expected, especially to make Preview components work correctly in this Next environment. There are a bit of trickeries here and there so that CSS files are handled correctly, so that image dimensions are defined, and other things. I'm happy to discuss this live if you prefer @lebaudantoine, no need to spend time on your own on this very boilerplaty-thing.

New things since last PR:

- commited the actual content (it was a mistake to ignore them in the last PR…)
- replaced hard-coded stuff from index/communs/legal/suivi/accessibilite pages with CMS content. Some content is still hard-coded and has not been replaced because it was a bit more time consuming to migrate them and I feel they are not the most important stuff, we could deal with that if needed
- the cms code is now easier to deal with in case we have more content types. The most time consuming thing is to build the decapcms json config for a content type

@todo : i18n!

### CODE REVIEW

File: content/content-pages/fr/suivi.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  ## Cookies déposés
+
+
+  Ce site  dépose un petit fichier texte (un « cookie ») sur votre ordinateur lorsque vous le consultez. Cela nous permet de mesurer le nombre de visites et de comprendre quelles sont les pages les plus consultées.
+
+
+  Vous pouvez vous opposer au suivi de votre navigation sur ce site web. Cela protégera votre vie privée, mais empêchera également le propriétaire d'apprendre de vos actions et de créer une meilleure expérience pour vous et les autres utilisateurs.
+
+
+  <iframe class="w-full" src="https://stats.data.gouv.fr/index.php?module=CoreAdminHome&amp;action=optOut&amp;language=fr"></iframe>
```
Comment (lebaudantoine) [inline content/content-pages/fr/suivi.yml:14]: The iframe is not showing !

**Review by lebaudantoine — COMMENTED** [review]
First functional review, let's discuss it tmr. I haven't red the code yet.

File: content/custom-pages/homepage.yml
[the remark this answers is not in the document]
Comment (lebaudantoine) [reply content/custom-pages/homepage.yml:1]: I think the slider / swiper component doesn't render properly

File: content/content-pages/fr/accessibilite.yml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  _Établie le 20 décembre 2023._
+
+
+  Cette déclaration d’accessibilité s’applique au site **lasuite.numerique.gouv.fr**.
+
+
+  ## État de conformité
+
+
+  **lasuite.numerique.gouv.fr** est non conforme avec le RGAA 4.1. Le site n’a pas encore été audité. Cependant, l'équipe s'efforce de créer un site accessible à tous en suivant les recommandations du RGAA.
```
[the remark this answers is not in the document]
Comment (lebaudantoine) [reply content/content-pages/fr/accessibilite.yml:21]: Discuss IRL: @manuhabitela will do some minor fixes, before we ask La Brigade for an audit.

File: content/custom-pages/homepage.yml
[the remark this answers is not in the document]
Comment (lebaudantoine) [reply content/custom-pages/homepage.yml:1]: Discussed IRL: Feels heavy in the CMS ui, it splits the content in multiple blocks.

File: README.md [hunk 3]
```diff
@@ -24,6 +30,22 @@ Open [http://localhost:3000](http://localhost:3000) with your browser to see the
 
 To learn more about Next.js, take a look at the [Next.js Documentation](https://nextjs.org/docs).
 
+## Authoring content
```
[the remark this answers is not in the document]
Comment (manuhabitela) [reply README.md:33]: Should be good as the subset is rather big from the start, we can double check when translated content is added before enabling i18n.

File: content/content-pages/fr/suivi.yml (hunk 1 above)
Comment (manuhabitela) [reply content/content-pages/fr/suivi.yml:14]: woops, thanks

File: content/custom-pages/homepage.yml
[the remark this answers is not in the document]
Comment (manuhabitela) [reply content/custom-pages/homepage.yml:1]: it's weird you get this actually, there is custom css to have a preview working without Swiper initialized (conflits with nextjs js loading/decapcms preview in iframe)…

You should get something like this:

image

**Comment by manuhabitela** [comment]
Made the iframe work and updated the docs to be more explicit.

Your preview panel CSS issue on the homepage testimonies is weird, you shouldn't have it… If you can try again if the up-to-date branch that would be helpful!

Thanks @lebaudantoine

**Comment by manuhabitela** [comment]
Merged the i18n related branch in this before merging for production too. When we have at least the homepage translated we can enable i18n on the frontend by uncommenting locales in the nextjs config.

**Pull request merged**